tests/testthat/comparison_data/emissions/L251.ctrl.delete.csv

```diff
@@ -0,0 +1,5516 @@
+INPUT_TABLE
+Variable ID
+DelEmCtrl
+
```

Comment:
Also, there are a couple more csv files in the changeset that are too large for github to render the diffs on them.  They need to be checked to see that they have correct line endings and for correctness of these metadata lines.

Also, doesn't the data system require a standard set of metadata comments for input files?